### ISSUE TITLE
need help about personal pictures's link generated in ged

### ISSUE THREAD

**Issue by crokignol** [geneweb/geneweb#45]
Hello,
somebody could tell me how to have these personnal pictures's link in my .ged ? Real I need.
I use geneweb 7 /win 7 .(of course i didn't made choise to hide them by "don't extract individual picture")
thanks

**Comment by ipfix** [comment]
If by "having them in my ged" you mean if you create on another computer with this ged file a new database, I will have my picture, then, the answer is you won't.

The gedcom is a text file, and you cannot have media data in it.

**Comment by crokignol** [comment]
No, I mean that I need these picture's link in my GED for export on geneanet site. I know how is ged files, and i know than it's necessary to have these picture's links in the gedfile. I have a lot of individual pictures  in the "images" folder, so when the gedfile is generated , normaly, i must find these links in the gedfile. But it's not and I cann't export pictures's links

**Comment by ipfix** [comment]
By default, the `gwb2ged` exports all information, except if you specified the option `-nopicture` which, as the name suggests, does't not export the picture. In order to see the link of the picture, you must have specified an image url in your database, and it will be automatically exported by `gwb2ged`.

Does that answer your question ? Actually I'm sorry, I'm not sure I understood it.

**Comment by a2line** [comment]
Can you come on IRC please?

**Comment by crokignol** [comment]
tell me how

**Comment by a2line** [comment]
Nearly the same way you found it last time http://webchat.freenode.net/?channels=geneweb
